index.js

```diff
@@ -1 +1,5 @@
-// Write your code in this file!
+const currentUser = "Nihar Patel";
+const welcomeMessage = `Welcome to Flatbook, ${currentUser}!`;
+const excitedWelcomeMessage = welcomeMessage.toUpperCase();
+
+const shortGreeting = `Welcome, ${currentUser.slice(0,1)}!`;
```

Comment:
Another way you could do this:
```suggestion
const shortGreeting = `Welcome, ${currentUser[0]}!`;
```